sotodlib/coords/local.py

```diff
@@ -0,0 +1,458 @@
+import numpy as np
+from astropy import units as u
+
+
+'''
+Most of the functions are adapted from pymap3d, to include error calulations 
+'''
+
+def ellipsoid(model='WGS84'):
+
+    '''
+    Return the major and minor semiaxis given an ellipsoid model.
+    Returns:
+    - a: astropy.Quantity, semiaxis major in meter
+    - b: astropy.Quantity, semiaxis minor in meter
+    '''
+
+    if model == 'WGS84':
+        a = 6378137.0*u.meter         
+        b = 6356752.31424518*u.meter 
+
+        return a, b
+
+def check_quantity(val, unit):
+
+    if isinstance(val, u.Quantity):
+        return val.to(unit)
+    else:
+        return val * unit
+
+def ecef2lonlat(x, y, z, ell='WGS84'):
```

Comment:
Throughout the module you use the "lonlat" abbreviation (great) but you often return those fields in the order (lat, lon, alt).  How would you feel returning/accepting (lon, lat, alt) everywhere?  (Note that's a naturally right-handed coordinate system, which we like.)

Reply:
Done. Also every input now follows the convention Lon,Lat,Alt